metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -484,7 +489,15 @@ object UserConfiguration {
     val superMethodLensesEnabled =
       getBooleanKey("super-method-lenses-enabled").getOrElse(false)
     val showInferredType =
-      getStringKey("show-inferred-type")
```

Comment:
Do we need to use `getBooleanKey`, it seems that `getStringKey` was working even with boolean values?

Reply:
I thought about for a while, if `getStringKey` is working then we can leave it